wagtail_streamfield_migration_toolkit/utils.py

```diff
@@ -10,17 +10,21 @@ def __init__(self, *args, instance=None, revision=None, **kwargs):
         super().__init__(*args, **kwargs)
 
     def __str__(self):
+        message = ""
+        message += super().__str__()
+        if self.args:
+            message += "\n"
```

Comment:
I think it would be useful to have the user-supplied message as the last line in the string. That way the most specific information can be in the most prominent position. Any preference @jacobtoppm ?

Reply:
Yep, I think that's slightly nicer too.